src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_pro_users.py

```diff
@@ -31,6 +35,8 @@ def create_industrial_pro_users():
                 postal_code="{}100".format(departement_code),
                 public_name="PC Test Pro {} {}".format(departement_code, pro_count),
             )
+            users_by_name["pro{} {}".format(departement_code, pro_count)] = user
+            create_user_offerer(offerer=offerers[index], user=user)
```

Comment:
Est-ce qu'il ne manque pas un `repository.save()` du UserOfferer créé ?

Mais en fait, j'utililserais volontiers `pcapi.core.offers.factories.UserOffererFactory` :

```suggestion
            UserOffererFactory(offerer=offerers[index], user=user)
```

Cela éviterait d'ajouter encore une occurrence de `create_user_offerer()` qu'on espère bien remplacer définitivement, un jour, avec tous ses copains-copines de `pcapi.model_creators`.

Reply:
C'est le repository.save du user derrière qui sauvegarde en passant le user_offerer

Les factories ce n'est pas uniquement pour les tests ?

Reply:
À mon avis, les factories sont des bons candidats aussi pour le code de la sandbox, puisque ça consiste à générer des données.

Reply:
Done, si c'est bon pour toi je te laisse approve ? :D